svg-to-png.js

```diff
@@ -0,0 +1,28 @@
+var path = require( 'path' );
+var svg_to_png = require('svg-to-png');
+
+async function convert() {
+    await svg_to_png.convert(
+        path.resolve('skins/moono-lexicon/icons'),
+        path.resolve('ckeditor-dev/skins/moono-lexicon/icons/hidpi/'),
+        {
+            defaultWidth: '32px',
+            defaultHeight: '32px',
+        }
+    );
+
+    console.log('HIDPI PNG icons created');
+
+    await svg_to_png.convert(
+        path.resolve('skins/moono-lexicon/icons'),
+        path.resolve('ckeditor-dev/skins/moono-lexicon/icons/'),
+        {
+            defaultWidth: '16px',
+            defaultHeight: '16px',
+        }
+    );
+
+    console.log('PNG icons created');
+};
+
+convert();
```

Comment:
You shouldn't fire off an async function and then just walk away. Instead, something like this:

```js
convert()
    .catch(error => {
        console.log(error);
        process.exit(1);
    });
```

All of this to avoid an "unhandled promise rejection" error.

Once we have [top-level await](https://github.com/tc39/proposal-top-level-await) broadly available, this will be nicer:

```
try {
   await convert();
} catch (error) {
   console.log(error);
   process.exit(1);
}
```